netmap/doc.go

```diff
@@ -54,17 +54,10 @@ Key-value storage format:
    Container contract reference
  - 'balanceScriptHash' -> 20-byte script hash
    Balance contract reference
- - 'notary' -> bool
```

Comment:
a separate commit?

Reply:
overall ur right, but here i thought that it's not really need, i wanna introduce single initial commit when all storage models are described and actual, and then change them with corresponding changes in the code

:+1: if it's worth separate commit anyway, i'll do